### PR TITLE
Milestone 3: Contact page

### PR DESCRIPTION
**Pull request by GedeonTS**
# PORTFOLIO: Contact page
In this part we:
- created markup for our contact page
- added styling to our contact page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,20 +213,78 @@ <h2 class="heading1 hd2">About<br />Myself</h2>
...
+                  name="email"
+                  type="email"
+                  class="user-input"
+                  placeholder="johndoe@gmail.com"
+                  required
+                />
+              </label>
+              <label class="contact-message">
+                <textarea
+                  name="message"
```
Comment (Deepakdanger) [inline index.html]: - [x] Please don't use your own name as placeholder text, what is intended with placeholder text is for example "full name" in the name input and "email address" in the mail input field.

File: index.html [hunk 2]
```diff
@@ -213,20 +213,78 @@ <h2 class="heading1 hd2">About<br />Myself</h2>
...
+            <li>
+              <a href="#" class="liss1 c">Skills</a
+              ><i class="fa fa-angle-right"></i>
             </li>
-            <li><a href="#" class="liss1 b">Frameworks</a><i class="fa fa-angle-right"></i></li>
-            <li><a href="#" class="liss1 c">Skills</a><i class="fa fa-angle-right"></i></li>
           </ul>
         </section>
+        <section class="myself abt3">
+          <div class="footer">
```
Comment (Deepakdanger) [inline index.html]: - [x] Unnecessary use of `div` tag inside `section ` tag here, when there is no content between them, I recommend using `footer` tag for good use of semantic tags.

File: index.html [hunk 3]
```diff
@@ -213,20 +213,78 @@ <h2 class="heading1 hd2">About<br />Myself</h2>
...
+                <textarea
+                  name="message"
+                  class="message"
+                  maxlength="500"
+                  required
+                >
+Enter your message here</textarea
+                >
+              </label>
+              <button type="submit" class="btn1 btn10">Get in touch</button>
```
Comment (Deepakdanger) [inline index.html:284]: - [x] The button should have an active, hover, pressed design given in the requirement. This has to be implemented on all buttons i.e "Get in touch", "See project", "Get resume" etc. Let's use this.
-
159135042-a4d912aa-0c7b-49f9-b791-4ab834bff39b

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @GedeonTS, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 4]
```diff
@@ -390,3 +385,76 @@ main {
...
+
+.btn1:hover {
+  background-color: #6c79ee;
+  color: #fff;
+}
+
+.btn2:hover {
+  background-color: #6c79ee;
+  color: #fff;
+}
```
Comment (AnselemOdims) [inline index.css]: - [x] As pointed out by the previous reviewer, the button should have the other states apart from hover which are `enabled` `active` and `disabled`

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @GedeonTS , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [x] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @GedeonTS , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**